pkgs/applications/graphics/renderdoc/default.nix

```diff
@@ -1,74 +1,60 @@
 { stdenv, fetchFromGitHub, cmake, pkgconfig
-, qtbase, qtx11extras, qtsvg, makeWrapper, python3, bison
-, pcre, vulkan-loader, xorg, autoreconfHook
+, qtbase, qtx11extras, qtsvg, makeWrapper
+, vulkan-loader, xorg
+, python36, bison, pcre, automake, autoconf
 }:
-
 let
   custom_swig = stdenv.mkDerivation {
-    name = "renderdoc-custom-swig";
+    name = "patched-custom-swig.tar";
     src = fetchFromGitHub {
       owner = "baldurk";
       repo = "swig";
-      rev = "renderdoc-modified-1";
-      sha256 = "1whymd3vamwnp4jqfc9asls3dw9wsdi21xhm1d2a4vx9nql8if1x";
+      rev = "renderdoc-modified-5";
+      sha256 = "0ihrxbx56p5wn589fbbsns93fp91sypqdzfxdy7l7v9sf69a41mw";
     };
-
-    nativeBuildInputs = [ autoreconfHook pcre ];
-
-    autoreconfPhase = ''
+    phases = ["unpackPhase" "patchAndMove"];
```

Comment:
Setting phases directly considered harmful, you could use `patchPhase` for the shebangs and `installPhase` for the tar.